test/integration/enumerate_databases/enumerate_databases.test.ts

```diff
@@ -2,8 +2,15 @@ import { expect } from 'chai';
 
 import { AddUserOptions, MongoClient, MongoServerError } from '../../../src';
 
+const metadata = {
+  requires: {
+    mongodb: '>=4.0.6',
+    auth: 'enabled',
+    topology: ['single', 'replicaset', 'sharded']
+  }
+} as any;
```

Comment:
Let's add MongoDBMetadataUI to the global namespace, and you can `as MongoDBMetadataUI` to get some hinting